lib/resources/r4/encounter_patch.yaml

```diff
@@ -0,0 +1,75 @@
+---
+name: Encounter
+field_name_base_url: http://hl7.org/fhir/R4/encounter-definitions.html#Encounter
+operations:
+  - name: replace-period-start
+    path: /period/start
+    operation: replace
+    cardinality: 0..1
+    type: dateTime
+    url: http://hl7.org/fhir/datatypes-definitions.html#Period.start
+    description: The start time of the encounter.
+    example: |
+      [
+      	{
+      		"op": "replace",
+      		"path": "/period/start",
+      		"value":  "2015-09-01T00:00:00.000Z"
+      	}
+      ]
+    note: >
+      <ul>
+        <li>Providing a nil value would update the `Encounter.period.start` value to nil.</li>
+        <li>When `Encounter.period.start` value is provided, time component should be provided.</li>
+      </ul>
+
+  - name: replace-period-end
+    path: /period/end
+    operation: replace
+    cardinality: 0..1
+    type: dateTime
+    url: http://hl7.org/fhir/datatypes-definitions.html#Period.end
+    description: The end time of the encounter.
+    example: |
+      [
+      	{
+      		"op": "replace",
+      		"path": "/period/end",
+      		"value":  "2017-09-01T00:00:00.000Z"
+      	}
+      ]
+    note: >
+      <ul>
+        <li>Providing a nil value would update the `Encounter.period.end` value to nil.</li>
```

Comment:
`nil` is specific to ruby, in JSON it would be `null`

Reply:
Fixed. https://github.com/cerner/fhir.cerner.com/commit/b3be566e03125de4bdc242726080efeaaef55e65#diff-97d84edc87e609cdbf80d6ad20cbc681R43

Reply:
How about wording like: "When no value is provided the `Encounter.period.end` will be unset."
Was trying to avoid using `null`

Reply:
yeah makes sense.

Reply:
Fixed. https://github.com/cerner/fhir.cerner.com/pull/240/commits/2f088fe9f198e6cb9d48c1be55d513f5eb8c9f72#diff-97d84edc87e609cdbf80d6ad20cbc681R20